python/cugraph-dgl/cugraph_dgl/utils/test_utils.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
```

Comment:
Are these utils used for the tests for this package, or are they intended to be used by your user's tests (or both)?  I'm just asking since if these are strictly for the tests for this package, then it might be better to move this to the `cugraph-dgl/tests` folder and rename it just `utils.py`.  One minor issue with this is that `pytest` may try to collect tests from here since it matches the standard file name for tests.

Reply:
Moved to `tests/utils.py`